sync/class.jetpack-sync-module-comments.php

```diff
@@ -24,6 +24,9 @@ public function init_listeners( $callable ) {
 		add_action( 'untrash_post_comments', $callable );
 		add_action( 'comment_approved_to_unapproved', $callable );
 		add_action( 'comment_unapproved_to_approved', $callable );
+		add_action( 'jetpack_modified_comment_contents', $callable );
```

Comment:
`add_action( 'jetpack_modified_comment_contents', $callable );` will need to be updated to 
`add_action( 'jetpack_modified_comment_contents', $callable, 10, 2 );`